{{cookiecutter.github_repository}}/provisioner/roles/project_data/defaults/main.yml

```diff
@@ -20,4 +20,9 @@ uwsgi_pid_file: "/tmp/uwsgi-{{ project_namespace }}.pid"
 
 uwsgi_log_dir: /var/log/uwsgi
 uwsgi_log_file: "{{ uwsgi_log_dir }}/{{ project_namespace }}.log"
+
+base_ubuntu:
+  common:
+    apt_packages:
+      - graphviz
```

Comment:
You would still want to add this to `tasks/main.yml` right? Skip the defaults and have just a task in tasks that installs this dependency.

Reply:
Done